src/Design.cs

```diff
@@ -625,6 +625,8 @@ private IEnumerable<Property> LoadDesignableProperties()
         yield return this.CreateProperty(nameof(this.View.X));
         yield return this.CreateProperty(nameof(this.View.Y));
 
+        yield return this.CreateSuppressedProperty(nameof(this.View.Visible), true);
```

Comment:
It will be a good choice to also add the `Enable` property because if it's false you can't get focus to it.
```cs
yield return this.CreateSuppressedProperty(nameof(this.View.Enable), true);`
```

Reply:
Well, it seems it's working and it's possible to change their properties, so forget what I said, please.